packages/suite/src/constants/suite/coinjoin.ts

```diff
@@ -1,14 +1,36 @@
 import { TranslationKey } from '@suite-common/intl-types';
-import { RoundPhase } from '@wallet-types/coinjoin';
+import { RoundPhase, SessionPhase } from '@trezor/coinjoin/src/enums';
```

Comment:
either lets export those enums from here like:
```
export { RoundPhase, SessionPhase } from '@trezor/coinjoin/src/enums';
```
an use them in suite like :
```
import { SessionPhase } from '@suite-constants/coinjoin'
```


or from export them from '@wallet-types/coinjoin' (RoundPhase is currently exported like that) 

i want to prevent importing syntax "from '@trezor/coinjoin/src/enums'" beause:
- if this file will change there will be a lot of places to change
- files imported like this are controversial and without proper explanation/comment someone who doesn't know the reason may "try to fix it"

Reply:
done